UofS-Pulse-Binfo/germ_summary
# Incorrect parent

This PR proposes major changes to the code-base of this module. The main change being the query considering the implementation of a 'caretaker/sitter' in references to a stock. In addition, this pr eliminated redundant query (ie. sample shown in tool tip text) and created system variables for frequently used ontologies (ie. is_selection_of).

## Metadata

 - Issue https://github.com/UofS-Pulse-Binfo/germ_summary/issues/13 Incorrect summary count per parent cross.




Documentation:

## Description



## Dependencies



## Testing?
1. Disable and uninstall this module.
2. Update files.
3. Enable module.

I have verified summary count of crosses where it is less than a hundred. Unknown by Unknown and culinaris and culinaris both returned count more than 700+, I've looked at some items but I could not confirm nor verify its correctness.
